okhttp/src/test/java/okhttp3/CacheTest.java

```diff
@@ -1940,8 +1940,10 @@ private RecordedRequest assertClientSuppliedCondition(MockResponse seed, String
     Response response3 = get(server.url("/a"));
     assertThat(response3.body().string()).isEqualTo("A");
     assertThat(response3.header("Allow")).isEqualTo("GET, HEAD");
-    assertThat((double) (response3.receivedResponseAtMillis() - t1)).isCloseTo(
-        (double) 0, offset(250.0));
+    // Since the received timestamp for response3 is the original timestamp from response2,
+    // we expect the difference from t2 to be greater than the sleep time(> 500ms).
+    assertThat((double) (t2 - response3.receivedResponseAtMillis())).isCloseTo(
```

Comment:
The comment doesn't match the logic.  Should the check be > 500ms?  This is 250 to 750.

Reply:
I'd like to remove the comment and change the test as I mentioned. (Let me add a new commit)
https://github.com/square/okhttp/pull/5552#issuecomment-542550095

Strictly specking, I should've done like this like you said?
```java
    assertThat(t2 - response3.receivedResponseAtMillis()).isGreaterThan(500);
    assertThat(t2 - response3.receivedResponseAtMillis()).isLessThan(750);
```